src/pages/login/login.js

```diff
@@ -40,6 +40,10 @@ const Login = () => {
 
   const dispatch = useDispatch();
 
+  useEffect(() => {
+    dispatch(resetState());
```

Comment:
Why do we need to reset user's state if we visit login page?

Reply:
if we had error on register page it would display on login as well